.vscode/launch.json

```diff
@@ -22,7 +22,18 @@
             "request": "launch",
             "name": "Debug Frontend: Launch Chrome against http://localhost:5173",
             "url": "http://localhost:5173",
-            "webRoot": "${workspaceFolder}/frontend"
+            "webRoot": "${workspaceFolder}/frontend",
+            // "runtimeExecutable": "/usr/bin/google-chrome"
```

Comment:
As described in #729, I am having trouble running the debug session on WSL.
I added this comment in case others encounter similar errors.

However, it did not solve my problem, so we could as well delete the comment and create a new PR to tackle #729.